recipes/sainsc/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
 
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
-  skip: True  # [osx or py<310]
+  number: 2
+  skip: True  # [(osx and not arm64) or py<310]
```

Comment:
We don't complie on ARM osx but nor intel OSX?

Reply:
Intel OSX didn't work and I couldn't figure out why so I thought better ARM than nothing

Reply:
Ok, interesting 🤔

Reply:
I am open for any suggestions or hints